distributed-databases/pinot/variables.tf

```diff
@@ -0,0 +1,34 @@
+variable "name" {
+  description = "Name of the VPC and EKS Cluster"
+  type        = string
+  default     = "pinot-on-eks"
+}
+
+variable "region" {
+  description = "region"
+  type        = string
+}
+
+variable "eks_cluster_version" {
+  description = "EKS Cluster version"
+  type        = string
+  default     = "1.25"
```

Comment:
did you try with latest EKS version?

Reply:
Did some preliminary testing with 1.28 and seems to be working. Don't see anything specific to k8s version [here](https://docs.pinot.apache.org/operators/tutorials/deployment-pinot-on-kubernetes) or [here](https://docs.pinot.apache.org/basics/getting-started/public-cloud-examples/aws-quickstart) also no reported issues on k8s version.